### PR TITLE
Update logging for ChildDetectors in Analysis and fixing accessibility bug - 3737201

### PR DESCRIPTION
**Pull request by puneetg1983**

### CODE REVIEW

File: AngularApp/projects/app-service-diagnostics/src/app/shared/components/collapsible-list/collapsible-list.component.scss [hunk 1]
```diff
@@ -38,4 +37,7 @@
 }
 .list-item.disabled:hover{
     text-decoration: none;
+}
+.list-item-title{
+    padding-left:10px;
```
Comment (dagrooms52) [inline AngularApp/projects/app-service-diagnostics/src/app/shared/components/collapsible-list/collapsible-list.component.scss]: I added bootstrap 4 spacing helpers, so `class="pl-3"` would apply similar padding without needing to create a class

File: AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts [hunk 2]
```diff
@@ -85,6 +85,15 @@ export class DetectorListAnalysisComponent extends DataRenderBaseComponent imple
           if (this.detectorId !== "") {
             let currentDetector = detectorList.find(detector => detector.id == this.detectorId)
             this.detectorName = currentDetector.name;
+            return;
```
Comment (dagrooms52) [inline AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts]: The original version let this method continue, is this supposed to exit without processing the `detectorList`?

Comment (puneetg1983) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts]: Yes, when we have a detector in the URL, we need not process all the child detectors

File: AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts [hunk 3]
```diff
@@ -85,6 +85,15 @@ export class DetectorListAnalysisComponent extends DataRenderBaseComponent imple
...
+          } else {
+            this.detectorEventProperties = {
+              'StartTime': String(this._detectorControl.startTime),
+              'EndTime': String(this._detectorControl.endTime),
+              'DetectorId': this.analysisId,
+              'ParentDetectorId': "",
+              'Url': window.location.href
```
Comment (cindylovescoding) [inline AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts]: Can we also add the detectorType to tell if it's analysis or detector?

Comment (puneetg1983) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts]: We can add it. Does it have any benefit if we add or don't add it?

Comment (puneetg1983) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts]: not opposing the ask, just trying to understand how it helps :-)

Comment (cindylovescoding) [reply AngularApp/projects/diagnostic-data/src/lib/components/detector-list-analysis/detector-list-analysis.component.ts]: Just want to make sure we don't mix the usage analysis for some of the detector type and analysis type with the same name. But it is fine for now.

**Pull request merged**